resdk/tests/functional/analysis/e2e_chip_seq.py

```diff
@@ -0,0 +1,184 @@
+# pylint: disable=missing-docstring
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from resdk.tests.functional.base import BaseResdkFunctionalTest
+
+
+class TestChipSeq(BaseResdkFunctionalTest):
+
+    def test_macs(self):
```

Comment:
Same as above, add note to the CHANGELOG about added E2E tests for `macs` and `rose`.